### ISSUE TITLE
xcode 7.3 support for Carthage

### ISSUE THREAD

**Issue by howie-misfit** [IFTTT/RazzleDazzle#22]
Can you help to pre-build binaries for Xcode 7.3 and Swift 2.2?

When i use 

> carthage update RazzleDazzle --no-use-binaries

i always get this error: 

> The following build commands failed:
>     CompileSwift normal x86_64 /Users/howie/Git_Repos/Work/Home_iOS/Leo/Carthage/Checkouts/RazzleDazzle/Example/Carthage/Checkouts/Quick/Externals/Nimble/Nimble/Adapters/NimbleXCTestHandler.swift
>     CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
> (2 failures)

**Comment by Mortenest** [comment]
Same here! Any fixes yet?

**Comment by nuudles** [comment]
@howie-misfit @Mortenest I was able to get it compiling by cloning the repository, updating the `Example/Cartfile.private` file with the newest versions for Quick and Nimble:

```
github "Quick/Nimble" "v4.0.0"
github "Quick/Quick" "v0.9.2"
```

**Comment by maxmeyers** [comment]
@nuudles Would you mind sending opening a pull request with that change?

**Comment by nuudles** [comment]
@maxmeyers PR #23 created.

**Comment by maxmeyers** [comment]
Great. This is fixed in 0.1.3, so I'll close this issue.
